components/use-cases/use-cases.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Use Cases
+permalink: /components/use-cases/
+---
+
+* TOC
+{:toc}
+
+Data templates are pre-defined sets of [component configurations](/components/). The individual configurations are connected 
```

Comment:
title je use cases a pak se dál mluví o data templates - jako vidím, že v UI je to úplně stejně dementní, ale to se fakt musíme rozhodnout jak se tomu bude říkat

Reply:
Use cases a data templates prý není úplně to samý. Monika říká, že use case je skupina šablon a šablona je konkrétní varianta use case. Třeba ecommerce je use case, ale bude se skládat ze 30 šablon. Teď je jich tam 7.

Reply:
oukej, ale pak to teda musíme nějak vysvětlit tady (i v UI) (pokud teda userovi na tomhle konceptu vůbec nějak záleží), ale v každým případě nemůžeme něco nadepsat "use case" a pak mluvit o "data templates".

Reply:
Jo a jestli jsem to nepsal, tak bych to celý klidně dal do toplevel - když je to tak v produktu, tak ať je to tak i tady

Reply:
ad navigace - hotovo

ostatni @MonikaFeigler

Reply:
máme svolení přejmenovat na Templates

Reply:
@hhanova